### ISSUE TITLE
Review doc/Language/classtut.pod6 headings

### ISSUE THREAD

**Issue by coke** [Raku/doc#4125]
post #4047, need to make sure that xt/search-categories.t passes and that the headings make sense going into the new category listing.

**Comment by coke** [comment]
```
$ xt/search-categories.t doc/Language/classtut.pod6 2>&1 | grep 'not ok'
    not ok 58 - Correct dimension (2) for a search anchor 'static' doc/Language/classtut.pod6
    not ok 63 - 「traits」is a valid category
    not ok 66 - 「traits」is a valid category
    not ok 105 - 「classes」is a valid category
not ok 1 - doc/Language/classtut.pod6
```

**Comment by tbrowder** [comment]
That should be fixed with PR #4158.

**Comment by coke** [comment]
Fixed by @tbrowder in 7e91a0018
